Consul.Test/KVTest.cs

```diff
@@ -354,16 +354,18 @@ public async Task KV_Keys_DeleteRecurse()
             Assert.Null(pairs.Response);
         }
 
-        [Fact]
-        public async Task KV_AcquireRelease()
+        [Theory]
+        [InlineData("test")]
+        [InlineData("test2")]
+        public async Task KV_AcquireRelease(string stringValue)
```

Comment:
I think that the test parameters here don't add any value. My intention in the previous comment was that we use different text values for `Acquire` and for `Release`, so we can check what value is being written by `Release`.

Reply:
> Not exactly sure if I understand what the difference would be.
I want to assert that `Release` operation releases the KV but also writes a new value.